notesapi/v1/views.py

```diff
@@ -275,20 +283,36 @@ def post(self, *args, **kwargs):  # pylint: disable=unused-argument
 
         Returns 400 request if bad payload is sent or it was empty object.
         """
-        if 'id' in self.request.data:
+        if not self.request.data or 'id' in self.request.data:
```

Comment:
this is needed for the tests or even real cases where request data is empty. As we first check for notes count before creating a note. Validation error raises when we create note.